src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala

```diff
@@ -0,0 +1,495 @@
+(@arg(flag='i', doc="Input text file of pileups generated by SvPileup") input: FilePath,
+ @arg(flag='b', doc="Bam file for allele frequency analysis. Must be the same file that was used as input to SvPileup.")
+ bam: Option[PathToBam] = None,
+ @arg(flag='f', doc="If bam file is provided: distance upstream and downstream of aggregated breakpoint regions to " +
+   "search for mapped templates that overlap breakends") flank: Int = 200,
+ @arg(doc="If bam file is provided: minimum total number of templates supporting an aggregated breakpoint to report " +
+   "allele frequency. Supports speed improvement by avoiding querying and iterating over huge read pileups that " +
+   "contain insufficient support for a breakpoint to be considered interesting.") minTotal: Int = 10,
+ @arg(doc="If bam file is provided: minimum allele frequency to report. Supports speed improvement by " +
+   "avoiding iterating over huge read pileups that contain insufficient support for a breakpoint to be considered " +
+   "interesting.") minFrequency: Double = 0.001,
+ @arg(flag='t', doc="Optional bed file of target regions") targetsBed: Option[FilePath] = None,
+ @arg(flag='o', doc="Output file") output: FilePath,
+ @arg(flag='d', doc="Distance threshold below which to cluster breakpoints") maxDist: Int = 10,
+) extends SvTool with LazyLogging {
+
+  Io.assertReadable(input)
+  Io.assertCanWriteFile(output)
+
+  override def execute(): Unit = {
+
+    // Read pileups from input file
+    val pileups: Seq[BreakpointPileup] = Metric.read[BreakpointPileup](input)
+    logger.info(f"Read ${pileups.length} pileups from file $input")
+
+    // Open bam reader
+    val samSource: Option[SamSource] = bam match {
+      case None => None
+      case Some(path) => Some(SamSource(path))
+    }
+
+    // Read targets from bed file and group by contig
+    val targets: Option[Map[String, Set[Interval]]] = targetsBed match {
+      case None => None
+      case Some(bed) =>
+        val bedReader = AbstractFeatureReader.getFeatureReader(bed.toString, new BEDCodec(), false)
+        val iter: CloseableTribbleIterator[BEDFeature] = bedReader.iterator()
+        val contigToTargets = mutable.HashMap[String, Set[Interval]]()
+        iter.forEach(feature => {
+          val contig = feature.getContig
+          val interval = new Interval(contig, feature.getStart, feature.getEnd)
+          contigToTargets.put(contig, contigToTargets.getOrElse(contig, new HashSet[Interval]()) + interval)
+        })
+        bedReader.close()
+        iter.close()
+        logger.info(f"Read ${contigToTargets.values.map(_.size).sum} targets on ${contigToTargets.keys.size} " +
+          f"contigs from file $bed")
+        Some(contigToTargets.toMap)
+    }
+
+    // Open output writer
+    val writer = Metric.writer[AggregatedBreakpointPileup](output)
+
+    // Group pileups by left and right contig and strand
+    pileups.groupMap(BreakpointContigOrientation(_))(identity).foreach { case (_, group) =>
```

Comment:
Why use `groupMap` with `identity` when `groupBy` is the same thing?  

```suggestion
    pileups.groupBy(BreakpointContigOrientation(_)).foreach { case (_, group) =>
```